src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -863,3 +863,24 @@ table.striped > tbody > tr > th {
         padding-right: 12px;
     }
 }
+
+pre.snippet {
+    background-color: #e8e8e8;
+    padding: 10px;
+    margin: 12px 0;
+    overflow: auto;
+    white-space: pre;
+}
+
+.italic {
+    font-style: italic;
+}
+
+.bold {
+    font-weight: bold;
+}
```

Comment:
Both the name and definition seem very global.  Maybe it would be better to restrict the effect of these names to within `pre.snippet`. @hns to advise?

Reply:
In commit 34b274c, I changed the CSS as suggested.